pkgs/applications/audio/furnace/default.nix

```diff
@@ -18,18 +18,22 @@
 , libjack2
 , withGUI ? true
 , Cocoa
+, portaudio
+, alsa-lib
+# Use GLES instead of GL
+, withGLES ? stdenv.hostPlatform.isAarch
```

Comment:
Why aarch-only?

Reply:
From personal experience with my own hardware, GLES support on ARM tends to be better than full GL support. Not sure what the situation is like on other drivers / SoCs / arches.